packages/frontend/src/components/Game/GameBoard/index.test.jsx

```diff
@@ -117,4 +110,79 @@ describe('GameBoard component', () => {
     await user.click(tile.container('tile-7').get());
     expect(tile.container('tile-7').get()).not.toHaveClass('faceUp');
   });
+
+  describe('player turn', () => {
+    it('does not award a point for a non match', async () => {
+      const tiles = createTilesFromPhotos(mockPhotos, { shuffle: false });
+
+      const initialGameState = produce(baseState.game, (draft) => {
+        draft.tiles = tiles;
+      });
+
+      const state = { ...baseState, game: initialGameState };
+
+      const { user, screen } = setupTests(GameBoard, { state });
+
+      const currentPlayerScore = screen.getByTestId('player-score-1');
+
+      expect(currentPlayerScore).toHaveTextContent('0');
+
+      await user.click(tile.container('tile-5').get());
+      await user.click(tile.container('tile-2').get());
+      act(() => jest.advanceTimersByTime(3000));
+
+      const updatedScore = screen.getByTestId('player-score-1');
+      expect(updatedScore).toHaveTextContent('0');
```

Comment:
If we wanted to get super organized, all the UI elements could be moved to the `ui` object, but there could be another ticket to make that migration.